e2e/vue3-vite-e2e/tests/component.spec.ts

```diff
@@ -24,53 +24,53 @@ describe('component e2e', () => {
     // Check file exists
     const componentName = names(component).className;
     checkFilesExist(
-      `apps/${app}/src/components/${componentName}.cy.ts`,
-      `apps/${app}/src/components/${componentName}.vue`
+      `apps/${app}/src/app/components/${componentName}.cy.ts`,
+      `apps/${app}/src/app/components/${componentName}.vue`
```

Comment:
Hi @sampullman 

It seems that components should be generated at `src/app/components` instead of `src/components` in an app.
So I make some changes to component generator as well as the related tests.

![Screenshot 2023-01-03 at 1 35 46 PM](https://user-images.githubusercontent.com/59676941/210304770-f77bdc08-9f48-410d-9788-a15d3ddf25d0.png)